src/main/scala/mesosphere/marathon/core/CoreGuiceModule.scala

```diff
@@ -45,6 +56,28 @@ class CoreGuiceModule extends AbstractModule {
   @Provides @Singleton
   final def appInfoService(appInfoModule: AppInfoModule): AppInfoService = appInfoModule.appInfoService
 
+  @Provides @Singleton
+  def taskStatusUpdateSteps(
+    notifyHealthCheckManagerStepImpl: NotifyHealthCheckManagerStepImpl,
+    acknowledgeTaskUpdateStepImpl: AcknowledgeTaskUpdateStepImpl,
+    notifyRateLimiterStepImpl: NotifyRateLimiterStepImpl,
+    emitUpdateStepImpl: EmitUpdateStepImpl,
+    postTaskStatusEventStepImpl: PostTaskStatusEventStepImpl,
+    scaleAppUpdateStepImpl: ScaleAppUpdateStepImpl,
+    notifyLaunchQueueStepImpl: NotifyLaunchQueueStepImpl,
+    updateTaskTrackerStepImpl: UpdateTaskTrackerStepImpl): Seq[TaskStatusUpdateStep] = {
+    Seq(
```

Comment:
Like that - very readable. Although would like to have 1 or 2 sentences documentation here, why this order is important.